tendrl/alerting/handlers/__init__.py

```diff
@@ -88,6 +91,9 @@ def update_alert(self):
                 )
             )
 
+    def classify_alert(self):
```

Comment:
In that case I feel `classify_alert` might not be very prominent name here. I am not able to get a better name though :) Think of something else.